src/Tribe/Template.php

```diff
@@ -767,6 +760,36 @@ public function template( $name, $context = [], $echo = true ) {
 		return $html;
 	}
 
+	/**
+	 * Includes a give PHP inside of a safe context.
+	 *
+	 * This method is required to prevent template files messing with local variables used inside of the
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: Line exceeds maximum limit of 100 characters; contains 104 characters

<sub>hash: 07b3267e7c3ced63cce54326e03068d5</sub>